spec/integration/indexer_integration_spec.rb

```diff
@@ -5,7 +5,11 @@
     it 'solr_doc has donor_tags_ssim field when <note displayLabel="Donor tags"> is in MODS' do
       r = Spotlight::Resources::Purl.new(url: 'https://purl.stanford.edu/vw282gv1740') # Feigenbaum PURL with donor tags
       solr_doc = r.to_solr.first
-      expect(solr_doc['donor_tags_ssim']).to eq ['Knowledge Systems Laboratory', 'medical applications', 'Publishing', 'Stanford', 'Stanford Computer Science Department']
+      expect(solr_doc['donor_tags_ssim']).to contain_exactly 'Knowledge Systems Laboratory',
+                                                             'medical applications',
+                                                             'Publishing',
+                                                             'Stanford',
+                                                             'Stanford Computer Science Department'
```

Comment:
it almost seems more worthwhile to rewrite this spec to a unit spec than to revise it here.